docs/example-scenario/serverless/microservices-with-container-apps-content.md

```diff
@@ -164,9 +164,9 @@ Security provides assurances against deliberate attacks and the abuse of your va
 
 - Environment variables: sensitive values can be securely stored at the application level.  When environment variables are changed, the container app will spawn a new revision.
 
-- Network security:  Backend services in the drone delivery app aren't exposed via external ingress. All requests are passed from the **Ingestion service** through the Azure Service Bus.
+- Network security:  Backend services in the drone delivery app aren't exposed via external ingress. All requests are passed from the **Ingestion service** through the Azure Service Bus. The Ingestion service is the only service configured with an external ingress. Azure Container Apps run in the context of an environment, which is supported by a virtual network (VNET). When you create an environment, you can provide a custom VNET, otherwise a VNET is automatically generated for you. Generated VNETs are inaccessible to you as they're created in Microsoft's tenant. To take full control over your VNET, provide an existing VNET to Container Apps as you create your environment. Do note in this example a VNET is automatically generated for you which means you can't add custom NSGs or similar. Additional details can be found [here](https://learn.microsoft.com/en-us/azure/container-apps/networking).
```

Comment:
```suggestion
- Network security:  To limit external access, only the Ingestion service is configured for external ingress. The backend services are accessible only through the internal virtual network (VNET) in the Container Apps environment. Incoming requests are passed from the Ingestion service to the backend services through the Azure Service Bus. When you create an environment, you can provide a custom VNET; otherwise, a VNET is automatically generated. Generated VNETs are inaccessible to you as they're created in Microsoft's tenant, so you cannot add Azure network security groups (NSGs). This example uses an automatically generated VNET. For more information, see [Networking architecture in Azure Container Apps](https://learn.microsoft.com/en-us/azure/container-apps/networking).
```
Applying the suggested change from Catherine Bundy